### ISSUE TITLE
Fish slice and ham items are inedible by carnivorous Origins

### ISSUE THREAD

**Issue by 7PkmnV** [newhoryzon/farmers-delight-fabric#72]
The raw/cooked cod slice, raw/cooked salmon slice, and normal/smoked ham items are not properly interpreted as meat. If playing with the Origins mod, players with the Carnivore power can't eat these items.

**Comment by Zifiv** [comment]
Hello @7PkmnV,

Thank you for reporting the integration issue with Origin. I need to check how Origin interprets a food as a meat.
I think it's because fish slice and ham is missing in some common tag.

**Comment by ryleu** [comment]
yep it's a tag. farmers delight has issues with some other tags to my knowledge, so it may be worth going through all of the #c namespace tags and doing those at the same time
